include/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ if(INSTALL_MBEDTLS_HEADERS)
 
     file(GLOB headers "mbedtls/*.h")
     file(GLOB psa_headers "psa/*.h")
+    file(GLOB psa_driver_headers "psa/drivers/*.h")
```

Comment:
The `psa` header file namespace is shared with all the PSA components. So its use is not solely a decision for PSA crypto.

This pull requests only introduces files matching `psa/drivers/crypto*`, which is enough namespacing to avoid conflicts with other PSA components. But what names will external drivers use? Should each external driver get its own subdirectory? Should external driver header names have a mandatory prefix like `crypto_driver_*`?

Reply:
Backed out the change introducing the new folder. It's just going to be delaying the driver work and can be done another time.